src/site.js

```diff
@@ -248,9 +248,21 @@ const createKibanaListener = (() => {
     window.addEventListener("message", (msg) => {
 
       if (msg.data.filter && msg.data.key) {
+
+        const iframe = document.querySelector('iframe')
+        const { scope } = iframe && iframe.dataset
```

Comment:
Doesn't this have to be handled in other places as well, e.g. in https://github.com/hbz/oerworldmap-ui/blob/087c2a801f7c9828927d4105c0ed78dbabc30bcb/src/site.js#L224?

Reply:
With the last commit this is solved, I add the scope in the other listeners.